src/style.css

```
@@ -0,0 +1,96 @@
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  padding: 30px 14px 15px;
+}
+
+.card__img {
+  width: 160px;
+  min-height: 134px;
+  align-items: center;
+  margin: 0 auto 45px;
+}
+
+.card__picture {
+  width: 160px;
+  height: 134px;
+  overflow: hidden;
+}
+
+.card__name {
+  display: inline-block;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+}
+
+.card__code {
+  display: inline-block;
+  font-size: 11px;
+  margin-bottom: 10px;
+}
+
+.card__about {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.card__font {
+  display: flex;
+  height: 15px;
+}
+
+.stars__star {
+  height: 15px;
+  width: 15px;
+  background-image: url("images/star.svg");
+  padding-left: 4px;
+  background-position: 10% 50%;
+  background-repeat: no-repeat;
+}
+
+.stars .stars__star:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.card-font {
+  font-size: 11px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.card__price-value {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+.card__btn {
+  display: inline-block;
+  padding: 12px 58px;
+  text-transform: uppercase;
+  text-decoration: none;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
+  background: #00acdc;
```

Этот цвет  #00acdc используется нескольно раз на странице, лучше огласить его как переменную
```suggestion
  parent {
    ...
    --color-blue: #00acdc;
  }
  ...
  child {
    ...
    background: var(--color-blue); 
    ...
  }
```